runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -113,12 +115,12 @@ private static int getFreshNamespaceIndex() {
 
     public MultiOutputExpandableTransform<InputT> withMultiOutputs() {
       return new MultiOutputExpandableTransform<>(
-          getUrn(), getPayload(), getEndpoint(), getNamespaceIndex());
+          getUrn(), getPayload(), getEndpoint(), getNamespaceIndex(), getOutputCoder());
```

Comment:
Not sure what this means. The coder should be used for all outputs? Likely we should disallow this to be non-null.

Reply:
Changed to a map instead of a singleton coder value.